contrib/build-push-ceph-container-imgs.sh

```diff
@@ -76,9 +76,14 @@ function install_docker {
     sudo apt-get update
     sudo env DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes docker-ce
   fi
-  sudo systemctl start docker
```

Comment:
how about just ``|| true`` on all of these?

long term, we should switch to buildah, probably, right?